transitions/generate_patient_id_on_patients.js

```diff
@@ -0,0 +1,11 @@
+var transitionUtils = require('./utils');
+
+module.exports = {
+  filter: function(doc) {
+    return doc.type === 'person' &&
```

Comment:
> Have we discussed potentially changing the IDs to person-$id

I have no idea what this means sorry.

> I like the idea of using the patient_id as a "short ID" or something that can be used for any person to do various things, that makes perfect sense.

So I think we should have a call when you're around next. That is already what they're used for. So I presume we're misunderstanding each other.

_Anyway_: the way we generate ids is definitely more complex than simply starting at 1 and heading on up. I don't think it's safe to presume the complexity doesn't come with a purpose. Unfortunately it's not documented anywhere I'm aware of, so we'd need to be very careful here. We'd also need to support both, since you can't change the scheme mid-project.

It's also a completely separate ticket. I don't really want this ticket to be held back because of our interest in maybe making a separate but related part of the system nicer.